ui/src/stylesheets/specific/navigation.scss

```
@@ -20,17 +20,33 @@
 }
 
 .nav-components {
-  width: calc(300px - (2.2rem * 2));
   height: calc(100% - 25px);
-  margin-top: 25px;
 }
 
-.nav-component {
-  padding-left: 30px;
+.btn-back {
+  opacity: .6;
+  margin-bottom: 0.5rem;
+  line-height: 1.5rem;
+  margin-left: 1.3rem;
+  cursor: pointer;
+
+  &::before{
+    display: inline-block;
+    content: url(../images/icons/arrow_back.svg);
+    margin-right: 0.3em;
+    margin-left: -1.3rem;
+    width: 1rem;
+  }
+}
+
+.nav-heading {
+  font-size: 1.3rem;
+  font-weight: var(--weight-medium);
+  font-family: var(--heading);
+  margin: .75rem 0;
 }
 
 .nav-component-active {
```

I think we have to add a minimum height here. It's causing the version selector to be cut off when the height is too short (see contract loader for example).

```suggestion
.nav-component-active {
  min-height: 100%;
```